hnn_core/gui/_viz_manager.py

```diff
@@ -99,6 +99,35 @@ def plot_type_coupled_change(new_plot_type, target_data_selection):
         target_data_selection.disabled = False
 
 
+def unlink_relink(attribute: str):
+    """
+    Wrapper function to unlink widgets to perform edits and re-link them on
```

Comment:
Check PEP 257 convention ... need a one-line description first ... longer description may follow later.

Reply:
"Wrapper function" as in decorator, right?